src/identities/privatekey.ts

```diff
@@ -84,6 +84,21 @@ export default class PrivateKey extends Identity implements IdentityInterface {
     this.scriptPubKey = p2wpkh.output!;
   }
 
+  private getAddress(): AddressInterface {
+    return {
+      confidentialAddress: this.confidentialAddress,
+      blindingPrivateKey: this.blindingPrivateKey,
+    };
+  }
+
+  getNextAddress(): AddressInterface {
+    return this.getAddress();
+  }
+
+  getNextChangeAddress(): AddressInterface {
```

Comment:
change address means internal chain in the derivation path

https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#Change